airflow/providers/google/cloud/hooks/cloud_sql.py

```diff
@@ -270,19 +263,18 @@ def patch_database(
         :type project_id: str
         :return: None
         """
-        if not project_id:
-            raise ValueError("The project_id should be set")
-        response = self.get_conn().databases().patch(  # pylint: disable=no-member
+        response = self.get_conn().databases().patch(  # noqa # pylint: disable=no-member
             project=project_id,
             instance=instance,
             database=database,
             body=body
         ).execute(num_retries=self.num_retries)
         operation_name = response["name"]
-        self._wait_for_operation_to_complete(project_id=project_id,
+        self._wait_for_operation_to_complete(project_id=project_id,  # type:ignore
                                              operation_name=operation_name)
 
     @CloudBaseHook.fallback_to_default_project_id
+    @CloudBaseHook.operation_in_progress_retry()
```

Comment:
Do we need this () brackets?

Reply:
yes. it returns a decorator  - this was copied from the quota_retry() and it is an easy way to get the tenacity default values for retry